alert.go

```diff
@@ -910,6 +916,31 @@ func (a *AlertNode) handlePost(post *pipeline.PostHandler, ad *AlertData) {
 	return
 }
 
+func (a *AlertNode) handleTcp(tcp *pipeline.TcpHandler, ad *AlertData) {
+	bodyBuffer := a.bufPool.Get().(*bytes.Buffer)
+	defer func() {
+		bodyBuffer.Reset()
+		a.bufPool.Put(bodyBuffer)
+	}()
+
+	err := json.NewEncoder(bodyBuffer).Encode(ad)
+	if err != nil {
+		a.logger.Println("E! failed to marshal alert data json", err)
+		return
+	}
+
+	conn, err := net.Dial("tcp", tcp.Address)
```

Comment:
Would it be possible to reuse the same TCP connection and use the Write method to send individual packets for each event? Not sure if that would break the server side.